charts/backstage/values.yaml

```
@@ -12,8 +16,12 @@ upstream:
     image:
       registry: quay.io
       repository: janus-idp/backstage-showcase
-      tag: latest
+      tag: next
     command: []
+    args:
+      - '--config'
+      - dynamic-plugins-root/app-config.dynamic-plugins.yaml
```

```suggestion
    command:
      - node
      - packages/backend
    args:
      - '--config'
      - 'app-config.yaml'
      - '--config'
      - 'app-config.example.yaml'
      - '--config'
      - 'app-config.example.production.yaml'
      - '--config'
      - dynamic-plugins-root/app-config.dynamic-plugins.yaml
```
As mentioned above, it would need to be something like this

---

Ad @tumido mentioned in [this comment](https://github.com/janus-idp/helm-backstage/pull/111/files#r1370738865) it doesn't seem necessary.